aws-javascript/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "${PROJECT}",
+    "name": "pulumi-project",
```

Comment:
I went back and forth whether these should use the name of the template, e.g. `aws-javascript` or just use the same generic value (e.g. `pulumi-project` or just `project`) across all templates. I don't have a strong preference either way. @lukehoban, do you have a preference?

This value doesn't really matter that much unless you're creating a library that you want to distribute on npm.

Reply:
Fixed merge conflict and decided to use the template names instead of generic `pulumi-project`. Pushed updated commit.